fakebook/src/app/component/profile-view/profile-view.component.html

```diff
@@ -14,6 +14,21 @@
   <h2 class="d-flex justify-content-center">
     {{ user?.firstName }} {{ user?.lastName }}
   </h2>
+  <div class="col text-center">
+    <button 
+    type="button" 
+    class="btn text-center" 
+    [ngClass]="followStatus ? 'btn-success' : 'btn-primary'" 
```

Comment:
just use [class binding](https://angular.io/guide/attribute-binding#binding-to-multiple-css-classes), it's improved to the point that we don't need ngClass directive anymore.

also, indent 2 spaces when breaking a tag across multiple lines

Reply:
changed to use class binding and fixed indentation